iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -311,6 +312,38 @@ public void rollbackAlterTable(org.apache.hadoop.hive.metastore.api.Table hmsTab
     }
   }
 
+  private void alterTableProperties(org.apache.hadoop.hive.metastore.api.Table hmsTable,
+      Map<String, String> contextProperties) {
+    Map<String, String> hmsTableParameters = hmsTable.getParameters();
+    Splitter splitter = Splitter.on(PROPERTIES_SEPARATOR);
+    UpdateProperties icebergUpdateProperties = icebergTable.updateProperties();
+    if (contextProperties.containsKey(SET_PROPERTIES)) {
+      splitter.splitToList(contextProperties.get(SET_PROPERTIES))
+          .forEach(k -> icebergUpdateProperties.set(k, hmsTableParameters.get(k)));
```

Comment:
I know it's not part of this PR's topic, but I just noticed: shouldn't we do the hms-iceberg property key translation here? instead of just `k` -> `HiveTableOperations.translateToIcebergProp(k)`

Reply:
You are right, we should do it.